### PR TITLE
Remove FormFieldControlWrapper; use layouts instead

### PR DESCRIPTION
**Pull request by libertymayc**
Remove FormFieldControlWrapper; use layouts instead

Aligns with the decision for Form Field Group concept to use layouts and give responsibility of accessibility and variants etc to user

### CODE REVIEW

**Comment by libertymayc** [comment]
Released as an rc. Is this a breaking change? Do we first deprecate the wrapper component?

**Review by Fercas123 — APPROVED** [review]

**Review by alycrys — APPROVED** [review]

File: packages/core/stories/form-field/form-field.stories.tsx [hunk 1]
```diff
@@ -184,67 +183,81 @@ export const MultiChild: ComponentStory<typeof FormField> = (props) => {
...
-          />
-          <Input
-            value={secondValue}
-            onChange={(e: ChangeEvent<HTMLInputElement>) => {
-              setSecondValue(e.target.value);
-            }}
-            placeholder="Multiplier"
-          />
-        </FormFieldControlWrapper>
+        <StackLayout gap={1} direction={"row"}>
```
Comment (ZarkoPernar) [inline packages/core/stories/form-field/form-field.stories.tsx:186]: How is this supposed to work when fieldset is the direct child, not the inputs?

**Comment by origami-z** [comment]
> Released as an rc. Is this a breaking change? Do we first deprecate the wrapper component?

Not a breaking change, but will likely mean more explanation needed for the changelog

File: packages/core/stories/form-field/form-field.stories.tsx (hunk 1 above)
Comment (libertymayc) [reply packages/core/stories/form-field/form-field.stories.tsx:186]: doh i wasn't thinking 🤕 changed it to role=group

**Review by alycrys — APPROVED** [review]

**Review by Fercas123 — APPROVED** [review]

**Review by Fercas123 — APPROVED** [review]

**Review by alycrys — APPROVED** [review]

**Pull request merged**